### DISCUSSION TITLE
beim reboot von RaspberryPi wo RaspberryMatic läuft werden weitere IPs angemeldet

### DISCUSSION THREAD

**Discussion by ArnaudFeld** [OpenCCU/OpenCCU#2456] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Wenn der Pi gebootet wird neben der AUX-ADDRESS und der IP-ADDRESS noch 1 weiteres "Device" am router angemeldet was einem die Netzwerk-Übersicht "vollkracht"

### Describe the behavior you expected

das sollte nicht passieren mit dem dritten "Device"

### Steps to reproduce the issue

1. RaspberryMatic im Docker installiert
2. "Problem" wird erst mit der Installation von RaspberryMatic im Docker ausgelöst. Extra auf einem frischen System ausgesetzt
3. rebootet man später irgendwann wird neben der AUX-ADDRESS und der IP-ADDRESS noch 1 weiteres "Device" am router angemeldet was einem die Netzwerk-Übersicht "vollkracht" mit der Zeit (bei mir .6 und .5 / dann kommt noch .90 dazu usw...)
4. bei jedem reboot kommt eins dazu

### What is the version this bug report is based on?

3.71.12.20230826

### Which base platform are you running?

oci (Open Container Infrastructure)

### Which HomeMatic/homematicIP radio module are you using?

HM-MOD-RPI-PCB

### Anything in the logs that might be useful for us?

```txt
Sep 29 16:12:51 Pi4-4GB avahi-daemon[431]: Joining mDNS multicast group on interface vetha14d37b.IPv6 with address fe80::3332:fafa:103:271.
Sep 29 16:12:51 Pi4-4GB avahi-daemon[431]: New relevant interface vetha14d37b.IPv6 for mDNS.
Sep 29 16:12:51 Pi4-4GB avahi-daemon[431]: Registering new address record for fe80::3332:fafa:103:271 on vetha14d37b.*.
Sep 29 16:12:51 Pi4-4GB dhcpcd[724]: veth06c352c: soliciting an IPv6 router
Sep 29 16:12:51 Pi4-4GB kernel: device eth0 entered promiscuous mode
Sep 29 16:12:51 Pi4-4GB dhcpcd[724]: veth1d5ada4: waiting for carrier
Sep 29 16:12:51 Pi4-4GB dhcpcd[724]: veth1d5ada4: carrier acquired
Sep 29 16:12:51 Pi4-4GB dhcpcd[724]: veth1d5ada4: IAID 07:f3:0e:24
Sep 29 16:12:51 Pi4-4GB dhcpcd[724]: veth1d5ada4: adding address fe80::21dd:daf0:87bf:f3d9
Sep 29 16:12:51 Pi4-4GB avahi-daemon[431]: Joining mDNS multicast group on interface veth1d5ada4.IPv6 with address fe80::21dd:daf0:87bf:f3d9.
Sep 29 16:12:51 Pi4-4GB avahi-daemon[431]: New relevant interface veth1d5ada4.IPv6 for mDNS.
Sep 29 16:12:51 Pi4-4GB avahi-daemon[431]: Registering new address record for fe80::21dd:daf0:87bf:f3d9 on veth1d5ada4.*.
Sep 29 16:12:52 Pi4-4GB dhcpcd[724]: veth1d5ada4: soliciting a DHCP lease
Sep 29 16:12:52 Pi4-4GB kernel: docker0: port 2(veth092add5) entered blocking state
Sep 29 16:12:52 Pi4-4GB kernel: docker0: port 2(veth092add5) entered disabled state
Sep 29 16:12:52 Pi4-4GB kernel: device veth092add5 entered promiscuous mode
Sep 29 16:12:52 Pi4-4GB kernel: docker0: port 2(veth092add5) entered blocking state
Sep 29 16:12:52 Pi4-4GB kernel: docker0: port 2(veth092add5) entered forwarding state
Sep 29 16:12:52 Pi4-4GB dhcpcd[724]: veth1d5ada4: soliciting an IPv6 router
Sep 29 16:12:52 Pi4-4GB dhcpcd[724]: vetha14d37b: soliciting an IPv6 router
Sep 29 16:12:52 Pi4-4GB dhcpcd[724]: veth092add5: IAID e5:bb:48:ac
Sep 29 16:12:52 Pi4-4GB dhcpcd[724]: veth092add5: adding address fe80::1b86:2f39:f917:8f2f
Sep 29 16:12:52 Pi4-4GB avahi-daemon[431]: Joining mDNS multicast group on interface veth092add5.IPv6 with address fe80::1b86:2f39:f917:8f2f.
Sep 29 16:12:52 Pi4-4GB avahi-daemon[431]: New relevant interface veth092add5.IPv6 for mDNS.
Sep 29 16:12:52 Pi4-4GB avahi-daemon[431]: Registering new address record for fe80::1b86:2f39:f917:8f2f on veth092add5.*.
Sep 29 16:12:52 Pi4-4GB dhcpcd[724]: vethd45c9c9: waiting for carrier
Sep 29 16:12:52 Pi4-4GB dhcpcd[724]: veth1d5ada4: offered 10.10.10.92 from 10.10.10.1
Sep 29 16:12:52 Pi4-4GB dhcpcd[724]: vethd45c9c9: carrier acquired
```

**Comment by jens-maus** [comment]
> ### Describe the issue you are experiencing
> Wenn der Pi gebootet wird neben der AUX-ADDRESS und der IP-ADDRESS noch 1 weiteres "Device" am router angemeldet was einem die Netzwerk-Übersicht "vollkracht"

Tut mir leid, keine Ahnung was du hier mit "was einem die Netzwerk-Übersicht 'vollkracht'" meinst. Liest sich für mich nach einem reinen lokalem Problem. Bitte erst einmal im Diskussionsteil erörtern. Auch ist das beigefügte Logfile nicht wirklich hilfreich bzw. aussagekräftig. Daher ab ins Diskussionsforum damit um das Problem erst einmal einzugrenzen.

**Comment by ArnaudFeld** [comment]
@jens-maus sorry mit "vollkracht" meinte ich das wenn der Pi über einen Zeitraum X ca. 10mal neugestartet wird (sei es wegen Updates, Stromunterbrechung usw...) wird jedesmal neben dem Container Host Aux-IP plötzlich ein weiteres "Gerät" im Router angezeigt (in meinem Fall z.B. 10.10.10.5 => Container IP, 10.10.10.6 => Container Host Aux-IP und eine weitere meldet sich an).

Wusste nicht genau welches Log man am besten nimmt und hatte im System recherchiert was so beim booten passiert.

Hab meinen zweiten frischen Pi mit deinem Docker-Script eingerichtet und der Fehler dort ist reproduzierbar. im Docker auf dem Test-Pi war nur RaspberryMatic installiert .

Bin sehr offen für Hinweise wo ich suchen kann im System
